.gitignore

```diff
@@ -1,3 +1,5 @@
 *.gem
 .guard-jruby-spec
-Gemfile.lock
\ No newline at end of file
+Gemfile.lock
+.idea/
```

Comment:
I'd rather not see `.idea` in the project .gitignore, as it's specific to your editor. Instead, this belongs in your global gitignore file. See https://help.github.com/articles/ignoring-files/ for setting this up.